eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/ClusteredVariantsRestController.java

```diff
@@ -135,5 +143,66 @@ public ClusteredVariantsRestController(
                                 .map(wrapper -> new AccessionResponseDTO<>(wrapper, SubmittedVariant::new))
                                 .collect(Collectors.toList());
     }
+
+    @ApiOperation(value = "Find a clustered variant (RS) by the identifying fields", notes = "This endpoint returns "
+            + "the clustered variant (RS) represented by a given identifier. For a description of the response, see "
+            + "https://github.com/EBIvariation/eva-accession/wiki/Import-accessions-from-dbSNP#clustered-variant-refsnp"
+            + "-or-rs")
+    @GetMapping(produces = "application/json")
+    public ResponseEntity getByIdFields(
+            @RequestParam(name = "assemblyId") String assembly,
```

Comment:
`@ApiParam` added for both parameters but agreed to add the examples in the description because with the springfox version we are using the `example` does not work and using `defaultValue` wouldn't be correct